BungeeCord-Patches/0057-Additional-DoS-mitigations.patch

```diff
@@ -71,15 +71,16 @@ index 37d28c01..01d35f41 100644
  
                  if ( in.isReadable() )
                  {
+-                    throw new BadPacketException( "Packet " + protocol + ":" + prot.getDirection() + "/" + packetId + " (" + packet.getClass().getSimpleName() + ") larger than expected, extra bytes: " + in.readableBytes() );
```

Comment:
The patch here somehow reverts the last upstream commit, and while it applies correctly I think this should be addressed here.
This line may not be missing or no exception will ever be thrown when in debug mode

Reply:
Thanks, fixed!